cloud/endagaweb/views/dashboard.py

```diff
@@ -286,6 +286,14 @@ def get(self, request, imsi=None):
                 'date')[0].date
         except IndexError:
             context['created'] = None
+        try:
+            # Get balance expire date for subscriber's first number.
+            number_details = Number.objects.filter(
```

Comment:
Lets tie balance to `Subscriber` since `Numbers` can move from one `Subscriber` to the next